lib/use-cases/SaveDropbox.js

```diff
@@ -20,32 +8,19 @@ module.exports = ({ gateways: { dropbox: dropboxes } }) => {
       throw new Error('Invalid dropbox');
     }
 
-    const {
-      description,
-      customerName,
-      customerDob,
-      customerEmail,
-      customerPhone,
-      customerReference,
-      customerNationalInsurance,
-      archived
-    } = fields;
+    const { description, firstName, dob, lastName, parentsEmail } = fields;
 
-    dropbox.description = validValueOrNull(description);
```

Comment:
We should probably keep this one in there as description is not a required field (so should be null in the db)